config.go

```diff
@@ -49,6 +53,8 @@ type config struct {
 	DcrdUser        string        `long:"dcrduser" ini-name:"dcrduser" description:"Username for dcrd RPC connections."`
 	DcrdPass        string        `long:"dcrdpass" ini-name:"dcrdpass" description:"Password for dcrd RPC connections."`
 	DcrdCert        string        `long:"dcrdcert" ini-name:"dcrdcert" description:"The dcrd RPC certificate file."`
+	LogDir          string        `long:"logdir" description:"Directory to log output."`
+	DataDir         string        `long:"datadir" description:"Directory to store data"`
```

Comment:
2 things:
- `log output`? If you opened the help info of a program and saw that a particular directory is the `directory to log output`, would you understand what that means?
- second description should end with a period. Also, `store data`, store which data?